### ISSUE TITLE
About easyocr training 

### ISSUE THREAD

**Issue by Ashingharoy1991** [JaidedAI/EasyOCR#1404]
I have created data for a custom dataset, where I have two folders: one for training and one for validation. However, when I start training, I get an error saying that index 1028 is not present. My training data has df.shape = (1028,), so the max index should be 1027. I have set data_filters = true in the training config file. I want to train a license plate recognition model using EasyOCR, where all characters are in capital letters, and only numbers from 0 to 9 are used, with no symbols. I need to know why I'm getting this error, and also how to fine-tune the model if I already have one.

**Comment by yashithalrao** [comment]
iirc youll need to have a labels.csv where 1st col name would be filename, and 2nd col name as words only then would it work
